README.md

```diff
@@ -48,11 +48,16 @@ Installation process for the setting up Wordpress on a local environment.
       git clone git@github.com:chapmanu/chap-press.git
       cd chap-press/public
       cp -v wp-config.php{-dist,}
-      wp core install --url=http://localhost:8222/ --title=chap-press --admin_user=chappress --admin_password=password --admin_email=chappress@gmail.com
 
       # The wp-config file already contains the database information to get started.
       # These are for local development only.
 
+- **Connect Wordpress to DB**
+	
+	While inside `/public` folder
+	  brew install wp-cli
+	  wp core install --url=http://localhost:8222/ --title=chap-press --admin_user=chappress --admin_password=password --admin_email=chappress@gmail.com
+
```

Comment:
Formatting needs to be fixed. See here:

- https://github.com/chapmanu/chap-press/tree/recreate-tests